src/gs/firts-touch/ever-wallet.md

```diff
@@ -0,0 +1,220 @@
+---
```

Comment:
This page duplicates src/develop/tutorial/ever-wallet.md

Reply:
https://github.com/everscale-org/docs/pull/297/commits/7d234bc32ac19f7fead257ff74a8c2dce4eac33f